### PR TITLE
Fix OS-ZOO CI on Alpine with GCC

### PR DESCRIPTION
**Pull request by t8m**
Apparently it does not like using sprintf(), let's replace with strcpy()

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: .github/workflows/os-zoo.yml [hunk 1]
```diff
@@ -130,7 +132,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branch: [openssl-3.0, openssl-3.1, master]
+        branch: [openssl-3.0, openssl-3.3, master]
```
Comment (nhorman) [inline .github/workflows/os-zoo.yml:135]: should we  be running these nightly on all active branches?

Comment (t8m) [reply .github/workflows/os-zoo.yml:135]: I'm not sure it really brings a value.

**Comment by t8m** [comment]
The Alpine fortify checks seem to be really broken, I'll do something else here - i.e. removing the warnings or something.

**Comment by t8m** [comment]
I am inclined to disable gcc build on edge as I am unable to make this work. The fortify headers are apparently broken there.

**Comment by t8m** [comment]
Hmm... they seem to be investigating - so we might want to wait a little https://github.com/jvoisin/fortify-headers/issues/62

**Comment by ncopa** [comment]
yeah, please wait a bit. If we cannot solve it within a few hours I'll revert.

**Comment by t8m** [comment]
Closing as the Alpine change was reverted.

**Pull request closed without merging**